src/linear_algebra/tensor_product.lean

```diff
@@ -875,6 +894,14 @@ by simp only [ltensor, rtensor, ← map_comp, id_comp, comp_id]
   (g'.ltensor _).comp (map f g) = map f (g'.comp g) :=
 by simp only [ltensor, rtensor, ← map_comp, id_comp, comp_id]
 
+variables {M}
+
+@[simp] lemma rtensor_pow (f : M →ₗ[R] M) (n : ℕ) : (f.rtensor N)^n = (f^n).rtensor N :=
+by { have h := map_pow f (id : N →ₗ[R] N) n, rwa id_pow at h, }
+
+@[simp] lemma ltensor_pow (f : N →ₗ[R] N) (n : ℕ) : (f.ltensor M)^n = (f^n).ltensor M :=
+by { have h := map_pow (id : M →ₗ[R] M) f n, rwa id_pow at h, }
```

Comment:
I'd be surprised if there isn't a one-line term-mode proof here.

Reply:
There may well be but the fact that `one_pow` (which `id_pow` uses) needs an inductive proof made me a bit doubtful. I'll have a go again now.

Reply:
I don't see it I'm afraid.